source/gui/__init__.py

```diff
@@ -30,6 +30,7 @@
 	# messageBox is accessed through `gui.messageBox` as opposed to `gui.message.messageBox` throughout NVDA,
```

Comment:
extra notes:

- we also need to deprecate and replace `runScriptModalDialog`. Because it creates a modal dialog that isn't tracked in `isModalMessageBoxActive`
- this PR also closes a bunch of issues listed in `13007`. Can these be added to the PR description so when this is merged they are also closed